templates/nodeovc/schema.capnp

```diff
@@ -0,0 +1,76 @@
+@0xd37ff48ad935931f;
+struct Schema {
+	# Description for the VM
+	description @0 :Text;
+
+	# Name of the VM
+	name @1 :Text;
+	
+	# OpenvCloud
+	openvcloud @2 :Text;
```

Comment:
As mentioned above, the machine should only know about it's VDC, and it then retrieves connection info from the vdc (the vdc should expose an action that return connection name (which itself get from the account))

it doesn't make any sense that we give it both the vdc and the connection

Reply:
Don't think you even need a action on the vdc that return the name. Usually instance name or guid of the service are used to create configuration instance. So as long as you know which vdc you belong, you should be able to get the configuration